crazyflie/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ find_package(sensor_msgs REQUIRED)
 find_package(crazyflie_interfaces REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(crazyswarm_application REQUIRED)
+find_package(motion_capture_tracking_interfaces REQUIRED)
```

Comment:
do not add in motion capture since we are not using or do not have motion capture module installed